### PR TITLE
Add style prop to Button

### PR DESCRIPTION
**Pull request by KarinaDavtyan**

### CODE REVIEW

File: packages/universal-components/src/Button/Button.stories.js [hunk 1]
```diff
@@ -62,8 +63,11 @@ storiesOf('Button', module)
     }
     const circled = boolean('circled', false);
     const label = text('label', 'Playground button');
+    const style = object('style', {});
```
Comment (JoseDuSV) [inline packages/universal-components/src/Button/Button.stories.js]: I think we can remove style object from Storybook, it doesn't seems to work (or I'm just trying values in wrong format 😄). We don't have it there for any other component.

Comment (KarinaDavtyan) [reply packages/universal-components/src/Button/Button.stories.js]: Yeah this one is tricky, on web it works like `{"borderRadius": 10}` but didnt manage to make it work on mobile

Comment (JoseDuSV) [reply packages/universal-components/src/Button/Button.stories.js]: Ok, so let just remove it. As I mentioned it's not available for any other styleable component.

Comment (KarinaDavtyan) [reply packages/universal-components/src/Button/Button.stories.js]: Ok, removed!

Comment (JoseDuSV) [reply packages/universal-components/src/Button/Button.stories.js]: Thanks

File: packages/universal-components/src/Button/Button.stories.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ import {
   select,
   text,
   number,
+  object,
```
Comment (JoseDuSV) [inline packages/universal-components/src/Button/Button.stories.js]: @KarinaDavtyan Just remove this one and we can merge it 🚀

Comment (KarinaDavtyan) [reply packages/universal-components/src/Button/Button.stories.js]: Done

File: packages/universal-components/src/Button/__tests__/index.test.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ const type = 'info';
 const testID = 'testID';
 const circled = false;
 const size = 'normal';
-
+const style = {};
```
Comment (RobinCsl) [inline packages/universal-components/src/Button/__tests__/index.test.js]: why was this change necessary? it seems like it's polluting the snapshots?

Comment (KarinaDavtyan) [reply packages/universal-components/src/Button/__tests__/index.test.js]: I assumed that all the props should be in the snapshot, should I remove it in separate PR?

Comment (JoseDuSV) [reply packages/universal-components/src/Button/__tests__/index.test.js]: It's that repeating question about what we are testing and how. One of the tests is for example `should have passed props` so it probably makes sense there (if we really want to test it).
And same for snapshots. Currently we are using `snapshotDiff` between Button with all props and without. Same as before, question is if it is really useful for something.

Comment (RobinCsl) [reply packages/universal-components/src/Button/__tests__/index.test.js]: Yes, we would need to sit down and decide what we believe is the best practice.
